crates/trippy/src/config.rs

```diff
@@ -1695,11 +1714,6 @@ mod tests {
         )?)
     }
 
-    fn remove_whitespace(mut s: String) -> String {
-        s.retain(|c| !c.is_whitespace());
-        s
-    }
```

Comment:
Now we have the `insta` fn as well I made the one in main::tests public to avoid repeating both of these

Reply:
yeah makes sense